packages/calcite-components/src/components/button/button.scss

```diff
@@ -1,4 +1,29 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-button-background-color: Specifies the background color of the component.
+ * @prop --calcite-button-text-color: Specifies the text and icon color of the component.
+ * @prop --calcite-button-box-shadow: Specifies the border color of the component.
+ * @prop --calcite-button-border-color: Specifies all border colors of the component.
+ * @prop --calcite-button-border-top-color: Specifies the top border color of the component.
+ * @prop --calcite-button-border-right-color: Specifies the right border color of the component.
+ * @prop --calcite-button-border-bottom-color: Specifies the bottom border color of the component.
+ * @prop --calcite-button-border-left-color: Specifies the left border color of the component.
+ *
+ */
+
 :host {
+  --calcite-button-background-color: var(--calcite-color-background);
+  --calcite-button-text-color: var(--calcite-color-text-1);
+  --calcite-button-box-shadow: var(--calcite-shadow-none);
+  --calcite-button-border-color: var(--calcite-color-transparent);
+  --calcite-button-border-top-color: var(--calcite-color-transparent);
```

Comment:
Do we want to expose these individual border color values? If so, should we use inline / block nomenclature? Unless this is needed for split-button, maybe these specific ones could be internal to begin?

Reply:
I'm not sure if they are 100% necessary but there is one spot in the button file where we are using `border-t-color-input` and `border-b-color-input` so I thought we might have wanted the option of individual border colors. I removed it for now because there's a lot going on but can revisit when there's some conclusion as to how we want to go about it.